Backend boilerplate

rlert opened the pull request.

rlert committed:
Created simple boilerplate for future webpage

rlert committed:
fixed boilerplate

Boilerplate styles and attributes are replicated from index.html file.

hthm00 reviewed (changes requested):
Excellent Work! Only small change is need. 

- Remove the test.js file and make changes directly to the index.js file
- boilerplate.ejs could be placed in folder views

rlert committed:
addressed change requests

- Removed test.js, transferred changes to index.js
- Moved boilerplate.ejs to views/pages/

rlert committed:
JSON file for view engine

hthm00 reviewed (approved):
Looks good!

rlert committed:
Resolved merge conflict

rlert committed:
Resolved merge conflict
